src/__tests__/components/navbarComponent/__test__/navbarComponent.test.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { BrowserRouter as Router } from "react-router-dom";
+import { shallow, mount } from "enzyme";
+import Navbar from "../../../../components/article/navbarComponent/Navbar";
+
+describe("Navbar not logged in test", () => {
+  const props = {
+    state: {
+      hidden: true,
+    },
+  };
+
+  it("should render without crashing", () => {
+    const wrapper = shallow(<Navbar {...props} />);
+    expect(wrapper).toMatchSnapshot();
+  });
+});
+
+describe("Navbar logged in test", () => {
+  beforeEach(() => {
+    localStorage.setItem("token", "token");
+  });
+
+  const props = {
+    state: {
+      hidden: false,
+    },
+    history: {
+      push: jest.fn(),
+    },
+  };
+
+  it("component should mount", () => {
+    const wrapper = mount(<Router><Navbar {...props} /></Router>);
+    console.log(wrapper.instance().props.children.props);
```

Comment:
Hey @wasibani-roy , You could eliminate this console log, when you get done testing on your local environment.